torch/_inductor/config.py

```diff
@@ -266,6 +266,9 @@ def autotune_remote_cache_default() -> Optional[bool]:
     "raise_comms",
 ]
 
+# enable operator reordering for peak memory optimization
+reorder_for_peak_memory = os.environ.get("TORCHINDUCTOR_REORDER_PEAK_MEMORY") == "1"
```

Comment:
we might want to add unit tests as well to prevent regression, e.g. example: https://github.com/pytorch/pytorch/blob/main/test/distributed/test_compute_comm_reordering.py

Reply:
added a unittest. Let me know if more detailed ones are needed.